capsules/src/spi.rs

```diff
@@ -44,6 +48,9 @@ struct SlaveApp {
     index: usize,
 }
 
+pub static mut SPI_WRITE_BUF: [u8; 1024] = [0; 1024];
+pub static mut SPI_READ_BUF: [u8; 1024] = [0; 1024];
+
```

Comment:
There should be no need for these buffers now that you have reverted to initializing in the component.

Reply:
Oops, of course. Sorry, I missed it in this patch while jumping between branches in the local repo. Thanks for notifying.